linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/configure_opensc_card_drivers/rule.yml

```diff
@@ -9,9 +9,8 @@ description: |-
     setting the smart card drivers in use by your organization helps to prevent
     users from using unauthorized smart cards. The default smart card driver for this
     profile is <tt>{{{ xccdf_value("var_smartcard_drivers") }}}</tt>.
-    To configure the OpenSC driver, edit the <tt>/etc/opensc-<i>ARCH</i>.conf</tt> (where
-    <i>ARCH</i> is the architecture of your operating system) file. Look for a
-    line similar to:
+    To configure the OpenSC driver, edit the <tt>/etc/opensc.conf</tt>.
+    Look for a line similar to:
     <pre># card_drivers = old, internal;</pre>
```

Comment:
Actually, since RHEL 7.7 we ship minimal `opensc.conf` which does not have these example lines. Even though your remediation scripts still work, the instructions here no longer work. It should probably say "add the following line into the file in the "app default" block so it will look like
```
app default {
   [...]
   card_drivers = {{{ xccdf_value("var_smartcard_drivers") }}};
}
```
This is the case also for the other occurrences modifying this file.

The other option might be to use the same steps as in the scripts using `opensc-tool --set-conf-entry`.

Reply:
Thanks!